lib/lita/adapters/irc/cinch_plugin.rb

```diff
@@ -32,7 +32,7 @@ def on_room_join(m)
           robot.trigger(
             :user_joined_room,
             user: user_by_nick(m.user.nick),
-            room: Lita::Room.find_by_name(m.channel.name),
+            room: Lita::Room::create_or_update(m.channel.name, name: m.channel.name),
```

Comment:
`.` should be preferred over `::` for method call syntax. I believe the latter is deprecated.